### PR TITLE
Adds Dell LTO5-140 (IBM model) to supported devices

### PR DESCRIPTION
**Pull request by sardaukar**
More info on this Dell (IBM rebrand) model: http://www.dell.com/tc/business/p/powervault-lto5-140/pd

### CODE REVIEW

**Comment by sardaukar** [comment]
Thanks for the help @mike-burns !

File: ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h [hunk 1]
```diff
@@ -43,6 +43,7 @@ const supported_device_type supported_devices[] = {
    { "ULTRIUM 7       ", drivefamily_lto,     "Quantum LTO7",      	drive_lto7		},
    { "LTO-5 HH        ", drivefamily_lto,     "TANDBERG DATA LTO5", drive_lto5 		},
    { "LTO-6 HH        ", drivefamily_lto,     "TANDBERG DATA LTO6",	drive_lto6		},
+   { "ULTRIUM-HH5     ", drivefamily_lto,     "DELL LTO5-140",      drive_lto5		},
```
Comment (retokromer) [inline ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h:46]: I cannot test this model now, but LGTM.

**Review by retokromer — APPROVED** [review]
@dericed Should I add a comment that it’s a modified file?

File: ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h (hunk 1 above)
Comment (sardaukar) [reply ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h:46]: Yes, maybe the comment does make sense! I've tested this at home, I have this tape drive.

Comment (retokromer) [reply ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h:46]: I was working on an `OpenLTFS` which should permit [LTOpers](https://github.com/amiaopensource/ltopers) to work on any manufacturer.

Comment (sardaukar) [reply ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h:46]: @retokromer does LTOpers work on Linux?

Comment (retokromer) [reply ltfs/src/tape_drivers/osx/ltotape/ltotape_supdevs.h:46]: @sardaukar On our setting (Ubuntu and Slackware with HP 6250), yes. And we are working on a Windows solution as well. It works already with little change on Windows Subsystem for Linux (Ubuntu).

**Comment by retokromer** [comment]
Thank you, @sardaukar!

**Pull request merged**